sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/DeviceInfoHelperTest.java

```diff
@@ -60,7 +63,16 @@ public void cleanWrapperSdk() {
     }
 
     @Test
-    public void getDeviceInfo() throws PackageManager.NameNotFoundException, DeviceInfoHelper.DeviceInfoException {
+    public void getDeviceInfoApi17() throws PackageManager.NameNotFoundException, DeviceInfoHelper.DeviceInfoException {
+        getDeviceInfo(17);
+    }
+
+    @Test
+    public void getDeviceInfoApi16() throws PackageManager.NameNotFoundException, DeviceInfoHelper.DeviceInfoException {
+        getDeviceInfo(16);
```

Comment:
The test does not verify that the API was switched over because the code is shared.
Should at least verify which string was used for get system service (and that the other was not used).